### PR TITLE
feat(docs): update popover page to use new layout

### PR DESCRIPTION
**Pull request by jorgemoya**
## What?

Update Popup page to use new layout.

## Screenshots/Screen Recordings
screencapture-localhost-3000-popover-2022-01-20-14_23_19

### CODE REVIEW

File: packages/docs/pages/popover.tsx [hunk 1]
```diff
@@ -39,30 +46,27 @@ const PopoverPage = () => {
...
 
-  return (
-    <>
-      <H1>Popover</H1>
-
-      <PageNavigation items={items} />
+      <Panel header="Do's and Don'ts" headerId="guidelines">
+        <GuidelinesTable
+          recommended={['Tie popup to click or hover events on elements.', 'Use short texts.']}
+          discouraged={['Always display popup.', 'Extended texts that disrupt the screen.']}
```
Comment (chanceaclark) [inline packages/docs/pages/popover.tsx:67]: What does `'Use short texts.'` and `'Extended texts that disrupt the screen.'` mean? Very confused by this 🤔

**Review by chanceaclark — APPROVED** [review]

File: packages/docs/pages/popover.tsx (hunk 1 above)
Comment (jorgemoya) [reply packages/docs/pages/popover.tsx:67]: I'm trying to represent that popups aren't supposed to be used for longer texts/paragraphs. Not sure how to word it correctly, or if it's relevant.

Comment (chanceaclark) [reply packages/docs/pages/popover.tsx:67]: Hmm... maybe like `Use concise textual phrases within.`? Not sure how to rephrase the last one though 🤔

Comment (jorgemoya) [reply packages/docs/pages/popover.tsx:67]: I'm just gonna remove it.

**Pull request merged**